README.md

```diff
@@ -1,6 +1,101 @@
-openmrs-module-openconceptlab
-=============================
+## OpenMRS Open Concept Lab module
 
-See project page at https://wiki.openmrs.org/x/XYA5B
+### Project page
 
-The current development board is at https://issues.openmrs.org/secure/RapidBoard.jspa?rapidView=93
+[OCL Subscription Module (Design Page)](https://wiki.openmrs.org/pages/viewpage.action?pageId=70877277)
+
+### Development board
+
+[Development board](https://issues.openmrs.org/secure/RapidBoard.jspa?rapidView=93)
+
+### Setup Development Environment for Open Concept Lab Module
+
+#### 1.Dependencies:
+- MySQL
+- Maven
+- JDK 1.8
+
+#### 2.Setup OpenMRS server
+
+Run this command to start OpenMRS server setup:
+````sh
+$ mvn openmrs-sdk:setup -DserverId=refapp -Dversion=2.3
+# Note: Pick default values for everything except MySQL username and password
+````
+
+If there are any issues with server setup, check out <b>[OpenMRS SDK documentation](https://wiki.openmrs.org/display/docs/OpenMRS+SDK)</b>
+
+#### 3.Installing Reference Application on server
```

Comment:
This step is not necessary. We just want to run the server we setup in the previous command. It will have all modules already.